src/components/ApplicationGroup/AppGroupRoute.tsx

```diff
@@ -2,21 +2,21 @@ import React, { useState } from 'react'
 import { URLS } from '../../config'
 import { Redirect, Route, Switch, useRouteMatch } from 'react-router-dom'
 import EnvironmentsList from './List/EnvironmentsList'
-import EnvironmentDetailsRoute from './EnvironmentDetailsRoute'
+import EnvironmentDetailsRoute from './AppGroupDetailsRoute'
 import { AppContext } from '../common'
 
-export default function EnvironmentsRoute() {
+export default function EnvironmentsRoute({isSuperAdmin}:{isSuperAdmin: boolean}) {
```

Comment:
Please move this to type and format the file

Reply:
done